### ISSUE TITLE
Octokit 'cannot load file'

### ISSUE THREAD

**Issue by brijshah** [unsplash/comment-on-pr#5]
When attempting to use this action to comment on a PR, the following stacktrace is presented:
```
/usr/local/bundle/gems/octokit-4.14.0/lib/octokit/middleware/follow_redirects.rb:14:in `<module:Middleware>': uninitialized constant Faraday::Error::ClientError (NameError)
Did you mean?  Faraday::ClientError
	from /usr/local/bundle/gems/octokit-4.14.0/lib/octokit/middleware/follow_redirects.rb:11:in `<module:Octokit>'
	from /usr/local/bundle/gems/octokit-4.14.0/lib/octokit/middleware/follow_redirects.rb:9:in `<top (required)>'
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /usr/local/bundle/gems/octokit-4.14.0/lib/octokit/default.rb:1:in `<top (required)>'
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /usr/local/bundle/gems/octokit-4.14.0/lib/octokit.rb:4:in `<top (required)>'
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `require'
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `rescue in require'
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:34:in `require'
	from /entrypoint.sh:4:in `<main>'
/usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- octokit (LoadError)
	from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /entrypoint.sh:4:in `<main>'
```

On subsequent runs of this workflow, the action works. So it seems to only happen on the first pass.

**Comment by steel** [comment]
There's an open issue in Octokit which is where this issue is coming from https://github.com/octokit/octokit.rb/issues/1155

**Comment by steel** [comment]
Manually bumping Faraday in the Dockerfile may resolve this issue if Octokit doesn't resolve the dependency itself. 

`bundle update faraday`

**Comment by steel** [comment]
Bumping didn't work so I just locked it at the previous version. PR https://github.com/unsplash/comment-on-pr/pull/6

**Comment by aaronklaassen** [comment]
This issue has been fixed in octokit 4.15.0
